Examples/MAX32655/BLE_FreeRTOS/dats_main.c

```diff
@@ -214,7 +214,7 @@ static struct {
 static dmSecLescOobCfg_t *datsOobCfg;
 
 /* Timer for trimming of the 32 kHz crystal */
-wsfTimer_t trimTimer;
+static wsfTimer_t* trimTimer = NULL;
```

Comment:
We were getting stack corruption if we used the statically allocated wsfTimer. Calling WsfTimerStop would cause subsequent problems.